src/tests/Microsoft.PowerFx.Connectors.Tests/IntellisenseTests.cs

```diff
@@ -96,8 +97,6 @@ public void ConnectorIntellisenseTest(int responseIndex, int queryIndex, string
  x-ms-user-agent: PowerFx/{PowerPlatformConnectorClient.Version}
 "
             };
-
-            Assert.Equal(expectedNetwork, networkTrace);
```

Comment:
reenable assert?

Reply:
Yes, this has been reenabled with McCall's recent PR to normalize the newlines